shell/components/__tests__/ApplicationCard.test.ts

```diff
@@ -0,0 +1,51 @@
+import { mount } from '@vue/test-utils';
+import ApplicationCard from '@shell/components/cards/ApplicationCard.vue';
+
+describe('component: CopyCode', () => {
+  const wrapper = mount(ApplicationCard, {
+    slots: {
+      cardIcon:         '<i class="icon icon-fw icon-epinio"></i>',
+      'top-left':       '<h1>Routes</h1> <ul><li> <a target="_blank" rel="noopener noreferrer nofollow"> route1 </a> </li> <li><a target="_blank" rel="noopener noreferrer nofollow"> route1 </a></li> </ul>',
+      'top-right':      '20 days ago',
+      resourcesCount: '<div> 2 Services</div> <div>3 Configs</div> <div>4 Environment variables</div>'
+    }
+  });
+
+  it('should have a populated card-icon slot', () => {
+    expect(wrapper.find('[data-testid="icon-slot-section"]').exists()).toBe(true);
+    expect(wrapper.find('[data-testid="icon-slot-section"]').find('i').exists()).toBe(true);
+    expect(wrapper.find('[data-testid="icon-slot-section"]').find('i').classes()).toContain('icon-epinio');
+  });
+
+  it('should have a populated top-left slot', () => {
+    const ele = wrapper.find('[data-testid="top-left-section"]');
+
+    expect(ele.find('ul').exists()).toBe(true);
```

Comment:
As the content has no constrain, the test is only testing that the markup you are writing is correct, which does not bring any use. It could be even just text or a simpler HTML element.

The idea is to check that the content is present and is in the correct section, regardless of what is passed. For this reason you could parameterize as defined above.